README.md

```diff
@@ -98,6 +98,64 @@ Replace ``address`` with the actual adress and replace myhomebrew with the name
 
 If no result is returned, make sure to build with the ``-g`` or``-g3`` option to make sure psp-addr2line knowns the function names and locations.
 
+## Debugging with GDB
+
+Sometimes debugging with the built in instruction level debugger is not enough, this is where
+PSPLINK's GDB Server can come in handy.
+
+Let's get started:
+
+### Preparation
+
+Prepare a separate terminal for `usbhostfs_pc`, `pspsh` and `psp-gdb`. All of them are needed to point the directory location to your `.prx` and `.elf` files located or the file you need to debug.
```

Comment:
```suggestion
Prepare a separate terminal for `usbhostfs_pc`, `pspsh` and `psp-gdb`. Open all of them in the directory in which your compiled `.prx` and `.elf` files are located.
```

Reply:
I'm not sure if this suggestion makes sense, since pspsh does noy need to be opened in a specific directory.